src/routes/missions.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Mission from '../components/missions/Mission';
+import getMissions from '../redux/missions/missionsThunk';
+
+const Missions = () => {
+  const dispatch = useDispatch();
+
+  const { missions } = useSelector((store) => store.missions);
+
+  useEffect(() => {
+    dispatch(getMissions());
+  }, [dispatch]);
+
```

Comment:
Well done for [implementing the rockets page ](https://github.com/Lornakaboro/space-missions/pull/35/files#diff-5121c185eec9f3fd4fffc411668cef164864b4ac706f0651d187ce2c98fd8ab7R8-R10)to load the data from the API only once, when the page loads. As specified in the [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/ac8c5ee08c46b1cef0a47e2e4f9adad1d26a5c00/group-project/project_space_travelers_hub.md?plain=1#L97), Kindly also modify the missions to load once.

Check out the illustration below:
https://user-images.githubusercontent.com/20554583/231821673-36d3db16-539e-4241-b2e6-dfcc0f564918.mp4